app/controllers/quotes_controller.rb

```diff
@@ -13,8 +13,12 @@ def new
 
   def create
     @quote = Quote.new(quote_params)
+
     if @quote.save
-      redirect_to quotes_path, notice: 'Quote was successfully created.'
+      respond_to do |format|
+        format.html { redirect_to quotes_path, notice: 'Quote was successfully created.' }
+        format.turbo_stream
+      end
```

Comment:
This action now supports _either_ `html` or `turbo_stream` formats. There's a new set of `turbo_stream` templates that accompany this change.